src/dug/cli.py

```diff
@@ -105,12 +106,11 @@ def search(args):
     dug = Dug(factory)
     # dug = Dug()
     response = dug.search(args.target, args.query, **args.kwargs)
-
     if not response:
        print ("No search results found")
     else:   
-       print(response)
-
+       jsonResponse = json.dumps(response, indent = 2)
+       print(jsonResponse)
```

Comment:
This works as expected. However I notice that if there are no results, then the code on line 110 doesn't get executed. It returns
```{
  "total_items": 0
}
```